docs/advanced_guide.rst

```diff
@@ -2,20 +2,41 @@
 
 Advanced Tutorials
 ==================
+This section contains examples and tutorials on more advanced topics, such as Multi Core computation, Custom operations, and more in depth applications
 
 .. toctree::
+   :caption: Examples
+   :maxdepth: 1
+
+   notebooks/neural_network_with_tfds_data
+   notebooks/Neural_Network_and_Data_Loading
+   notebooks/vmapped_log_probs
+
+.. toctree::
+   :caption: Multi-core and Multi-pod Computation
```

Comment:
Is POD an external name? i.e. something we use in cloud TPU docs?

Reply:
I've seen it used elsewhere (e.g. https://blogs.nvidia.com/blog/2021/03/05/what-is-a-cluster-pod/). Do you think multi-cluster or something like that would be more clear?

Reply:
Maybe "parallel and distributed computation"?

Reply:
I think we should ask Skye or James

Reply:
James recommended "Parallel Computation"